documentation.md

```diff
@@ -0,0 +1,35 @@
+# Documentation Policies
+
+## Documentation
+
+The project documentation will reside on the `wiki` page or/and in the` doc` folder of the project repository.
+
+When a new feature is added or a significant change is made. Firstly, a design proposal must be created and only once it is accepted and implemented will be added as part of the project documentation.
+
+## Design Proposals
+
+The design proposal is to make communication easier with team members and be clear about the idea and documentation. 
+
+A design proposal is necessary only when a new feature is added or a significant change is made. Therefore it is not necessary to do it for simple changes or bug fixes.
+
+To create a design proposal, you must create an `github issue` in the repository of the project and use the template below:
+
+```
+### Title
+A descriptive title
+
+### Overview
+High-level description
```

Comment:
Would be useful that the overview answer these three questions:
What?  Why? and How?

Reply:
Right! It is described in the proposal of the development process.